lib/susedistribution.pm

```diff
@@ -562,11 +563,16 @@ sub init_consoles {
         if (is_backend_s390x) {
             # expand the S390 params
             my $s390_params = get_var("S390_NETWORK_PARAMS");
-            my $s390_host = get_required_var('S390_HOST');
-            $s390_params =~ s,\@S390_HOST\@,$s390_host,g;
+            my $s390_guest_fqdn = get_required_var("ZVM_GUEST");
+            my ($s390_guest_hostname) = $s390_guest_fqdn =~ /(.*?)\..*$/;
+            my $s390_guest_subnetmask = get_required_var("ZVM_GUEST_SUBNETMASK");
```

Comment:
How about a sensible default value

```suggestion
            my $s390_guest_subnetmask = get_var('ZVM_GUEST_SUBNETMASK', 24);
```

Reply:
we'd rather be explicit here because it could lead to strange behavior later in the test if the default is used but differs from what is actually used in the network. Since it happens rather hidden in the background, an reviewer might not be able to spot this mistake otherwise (I wouldn't even trust myself to find it!).

Reply:
yeah, right.

Reply:
Ideally the default would still show up in settings? That's the first place I would check anyway.